ddmrp_report_part_flow_index/models/stock_buffer.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017-24 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class StockBuffer(models.Model):
+    _inherit = "stock.buffer"
+
+    flow_index_group_id = fields.Many2one(
+        "ddmrp.flow.index.group", string="Flow Index Group", readonly=True
+    )
+
+    def _calc_flow_index_group_id(self):
```

Comment:
I  just realized that this feature was added without any test case, could you add a small test for this?

Reply:
Test added!